src/NServiceBus.AcceptanceTests/Mutators/When_incoming_mutator_changes_message_type.cs

```diff
@@ -44,7 +44,8 @@ public class MessageMutator : IMutateIncomingMessages
             {
                 public Task MutateIncoming(MutateIncomingMessageContext context)
                 {
-                    context.Message = new NewMessage();
+                    var original = context.Message as OriginalMessage;
```

Comment:
why `as` and not a cast since the next line will blow anyway with `NullReferenceException`